pkg/conf/model.go

```diff
@@ -21,9 +21,24 @@ func NewContextConfig() *ContextConfig {
 
 // base structure that all server types need
 type NodeConfig struct {
+	// *** Required fields ***
+
 	Type NodeType `yaml:"type"`
+	// The delegate owner key can either be directly specified or an
+	// absolute path to a file on the HOST machine (not your local machine) containing the key
+	PrivateKey    string `yaml:"privateKey"`
+	Wallet        string `yaml:"wallet"`
+	RewardsWallet string `yaml:"rewardsWallet"`
+	// One of "current", "prerelease", or an audius-docker-compose git branch (for dev purposes)
+	// "current" corresponds to main adc branch
+	// "prelease" corresponds to stage
+	// defaults to "current" behavior if omitted
+	Version string `yaml:"version,omitempty"`
+
+	// *** Optional fields ***
 
-	// (Optional) Specify non-standard ports for http traffic
+	EthWallet string `yaml:"ethWallet,omitempty"`
```

Comment:
what's the significance of EthWallet over the other two wallets? aren't all three technically ethereum wallets?

Reply:
I honestly don't know, the creator node just has a separate config for "ethOwnerWallet" and I don't know why or how it differs from "spOwnerWallet". Could just be more legacy cruft that I'd be happy to get rid of.

Reply:
gotcha, it's probably the same but we can fix that later